Core/src/main/java/com/plotsquared/core/queue/LocationOffsetDelegateQueueCoordinator.java

```diff
@@ -87,11 +87,14 @@ public boolean setBlock(int x, int y, int z, @NonNull Pattern pattern) {
 
     @Override
     public boolean setBiome(int x, int z, @NonNull BiomeType biome) {
-        boolean result = true;
-        for (int y = 0; y < 256; y++) {
-            result &= this.setBiome(x, y, z, biome);
+        try {
+            if (canPlace[x - blockX][z - blockZ]) {
+                return super.setBiome(x, z, biome);
+            }
+        } catch (final Exception e) {
+            throw e;
```

Comment:
The exception just gets rethrown

Reply:
Matches every other method in the class

Reply:
Could be cleaned up then I guess, but we can do that separately to avoid even more diff